newIDE/app/src/EventsSheet/ParameterFields/FunctionParameterNameField.js

```diff
@@ -1,51 +1,87 @@
 // @flow
 import React from 'react';
-import GenericExpressionField from './GenericExpressionField';
+import { t } from '@lingui/macro';
 import {
   type ParameterFieldProps,
   type ParameterFieldInterface,
   type FieldFocusFunction,
 } from './ParameterFieldCommons';
-import { type ExpressionAutocompletion } from '../../ExpressionAutocompletion';
 import { enumerateParametersUsableInExpressions } from './EnumerateFunctionParameters';
+import SelectField, { type SelectFieldInterface } from '../../UI/SelectField';
+import SelectOption from '../../UI/SelectOption';
 
 export default React.forwardRef<ParameterFieldProps, ParameterFieldInterface>(
   function FunctionParameterNameField(props: ParameterFieldProps, ref) {
-    const field = React.useRef<?GenericExpressionField>(null);
+    const field = React.useRef<?SelectFieldInterface>(null);
```

Comment:
just to be sure, there's never a need to use an expression for a function parameter name?
(I don't think so, but just checking)

Reply:
Not really, and if users try it it would probably crash because the generator of these special instructions copy the expression directly in the code without translating it into JS.